node/src/chain_spec.rs

```diff
@@ -140,6 +141,7 @@ impl Alternative {
                             get_account_id_from_seed::<sr25519::Public>("Eve//stash"),
                             get_account_id_from_seed::<sr25519::Public>("Ferdie//stash"),
                         ],
+                        node_runtime::ProposalsConfigParameters::development(),
```

Comment:
Should it be "testing()" here?

Reply:
I considered that, but for network integration testing @gleb-urvanov needs 0 grace periods :)